pkg/config/config.go

```diff
@@ -204,7 +205,13 @@ func (c *Config) Merge(config Config) {
 	}
 
 	if config.SpacesAftertabs {
-		c.SpacesAftertabs = config.SpacesAftertabs
+		c.Logger.Warning("The configuration key `SpacesAftertabs` is deprecated. Use `SpacesAfterTabs` instead.")
+
+		c.SpacesAfterTabs = config.SpacesAftertabs
+	}
```

Comment:
* Should the value of `config.SpacesAftertabs` be stored in `c.SpacesAftertabs` ?
  Should I change the code to the following?
  ``` go
  if config.SpacesAftertabs {
  	c.Logger.Warning("The configuration key `SpacesAftertabs` is deprecated. Use `SpacesAfterTabs` instead.")

  	c.SpacesAftertabs = config.SpacesAftertabs
  	c.SpacesAfterTabs = config.SpacesAftertabs
  }
  ```

Reply:
* How can I make `editorconfig-checker` output a warning even if `SpacesAftertabs` is `false` ?
  The current code does't output the warning if `"SpacesAftertabs": false` is set in the configuration file.

  ``` console
  # "SpacesAftertabs": true
  $ cat .ecrc
  {
    "SpacesAftertabs": true
  }
  $ bin/ec
  The configuration key `SpacesAftertabs` is deprecated. Use `SpacesAfterTabs` instead.

  # "SpacesAftertabs": false
  $ sed -i 's/true/false/' .ecrc
  $ cat .ecrc
  {
    "SpacesAftertabs": false
  }
  $ bin/ec # No warning is output.
  ```

Reply:
I'm also fairly new to golang, so I just googled a bit and found this, which looks promising: https://stackoverflow.com/questions/31989595/detect-if-json-file-has-field

It should then mean, that after the config file has been decided you can check the `SpacesAftertabs` for not being a null pointer.

Reply:
Thanks for the help.
It worked.